src/Symfony/Component/VarDumper/Caster/MemcachedCaster.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class MemcachedCaster
+{
+    public static function castMemcached(\Memcached $c, array $a, Stub $stub, $isNested)
+    {
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'servers' => $c->getServerList(),
+            Caster::PREFIX_VIRTUAL.'options' => new EnumStub(
+                self::getMemcachedNonDefaultValueOptions($c)
+            ),
+        );
+
+        return $a;
+    }
+
+    public static function getMemcachedNonDefaultValueOptions(\Memcached $c)
+    {
+        $defaultOptions = self::discoverDefaultMemcachedOptions();
+        $optionConstants = self::getMemcachedOptionConstants();
```

Comment:
since both these values are static, what about caching them in static properties?
`self::$optionConstants = self::$optionConstants ?? self::getMemcachedOptionConstants();`, etc.